reduction/imaging_parameters.py

```diff
@@ -152,6 +156,14 @@
                            for contsub in ("","_contsub")
                           }
 
+
+line_parameters = {'n2hp'：
+                {'restfreq':'93.173700GHz',
+                'vlsr':'-18km/s', # the systematic lsr velocity
```

Comment:
because this only applies to one field, this should be nested deeper, e.g.:
```
line_parameters = {'G328': {'n2hp': {'vlsr': '-18km/s'}}}
```
like that - we'll need different vlsr and cubewidth for each field